sample/sample7-container/tsd.json

```diff
@@ -0,0 +1,22 @@
+{
```

Comment:
I am sorry but looks like I didnt pushed latest changes, but controllers.ts is already using typings instead of tsd (since tsd is deprecated) (1 week ago I pushed to npm latest version with typings), so you'll need to update this and use typings instead